### PR TITLE
[FLOC-1071] Document vagrant slave setup.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1071

### CODE REVIEW

**Comment by wallrj** [comment]
This seems to be more related to https://clusterhq.atlassian.net/browse/FLOC-180. Is it ready for a review @tomprince ?

**Comment by tomprince** [comment]
This is the configuration of the slave that is used for https://clusterhq.atlassian.net/browse/FLOC-1071

File: slave/vagrant/fabfile.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import yaml
+
+# WARNING: Although this code has been tested to produce a working slave
+# the production slave wasn't configured with this code.
+
+def configure_gsutil():
+    boto_config = FilePath(__file__).sibling('boto-config.in').getContent()
+    output = local('lpass show --notes "google-cert@build.clusterhq.com"',
+                   capture=True)
+    cert = yaml.safe_load(output.stdout)
```
Comment (wallrj) [inline slave/vagrant/fabfile.py]: I don't understand this part.
LastPass notes for that account contains the cert (in binary form) I think....not yaml.
Perhaps LastPass notes need updating or perhaps you've got a different LastPass entry for `google-cert@build.clusterhq.com` .
I'm looking at the one in Shared-dev.clusterhq.com

**Comment by wallrj** [comment]
Thanks @tomprince 

I think I understand what's going on here. This is how you're configuring some new "baremetal" build slaves, right? And these are able to run the vagrant (non-cloud) acceptance tests.

I would like to have seen a little more documentation...somewhere...after all, the linked ticket title is "Document vagrant slave setup". 

Is this the builder: http://build.clusterhq.com/buildslaves/fedora-vagrant-0 and if so, is it possible to at least update the "Slave information" where it currently says "Please put a description of this build host here"

And spotted what I think is a problem with the LastPass content (see above).

Do what you think is necessary and create followup issues as you see fit, then merge.

File: slave/vagrant/fabfile.py (hunk 1 above)
Comment (tomprince) [inline slave/vagrant/fabfile.py]: lastpass needed updating. lastpass (at least the command-line client) doesn't like embedded nulls, which the certificate has. Also, there are two pieces of data that need storing.

**Pull request merged**